src/App.css

```diff
@@ -1,38 +1,24 @@
-.App {
-  text-align: center;
-}
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap');
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+:root {
+  --font-family: 'Inter', sans-serif;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+html {
+  height: 100%;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+body {
+  height: 100%;
+  margin: 0;
+  font-size: 1.125rem;
+  font-family: var(--font-family);
 }
 
-.App-link {
-  color: #61dafb;
+input {
+  font-family: var(--font-family);
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+#root {
```

Comment:
Targeting all elements with an ID of "root"? Was that intentional. I believe you meant to apply this to the `:root`. But you shouldn't have to apply `height: 100%;` to the root. Do you need this?

Reply:
This was to apply 100% height to React's wrapper element that has an id of root. I used it to be able to center things vertically, like with the 100% on the html/body elements